plugins/node/opentelemetry-instrumentation-redis/src/utils.ts

```diff
@@ -125,6 +125,7 @@ export const getTracedInternalSendCommand = (
       );
     }
 
+    const originalContext = context.active();
```

Comment:
You could move this line down into the `if (originalCallback) {`.

Reply:
Thanks @Flarna, fixed.